src/pages/messages/TasksPage.tsx

```diff
@@ -291,11 +305,26 @@ const TasksPage = () => {
             />
           </StyledTicketSearchFormGroup>
         </NavigationListAccordion>
+
+        <BetaFunctionality>
+          <NavigationListAccordion
+            title={t('common.nvi')}
+            startIcon={<AdjustIcon sx={{ bgcolor: 'nvi.main', padding: '0.1rem' }} />}
+            accordionPath={UrlPathTemplate.TasksNvi}
+            onClick={() => {
+              if (!isOnNviCandidatesPage) {
+                setPage(1);
+              }
+            }}
+            dataTestId={dataTestId.tasksPage.nviAccordion}>
+            <></>
```

Comment:
Har denne noen hensikt?

```suggestion
            
```

Reply:
Kommer mer her i kommende PRs, og komponenten her krevde children, så da la jeg inn en tom tag i stedet for å sette children som optional for nå